### ISSUE TITLE
Inconsistent sync on Pixel 2XL

### ISSUE THREAD

**Issue by JoeT1969** [k9mail/k-9#3282]
Android 8.1
K-9 v5.403

I have three POP accounts setup, with Always Sync on every 60 minutes.  It worked fine on my Motorola Turbo, but not on my Pixel 2 XL.
It will show either Sync Disabled randomly throughout the day, or say Next Sync 45 min. ago, or a random time in the past.  Sometimes if I pick up the phone and unlock the screen, it'll start to sync.  Or it won't sync until I open K-9Mail.  During the day the phone is on cellular only, and at night it's on WiFi only.
I have to manually force downloading mail 90% of the time.
The new mail notification doesn't always clear, same as it didnt on the Motorola.

**Comment by lldaedalusll** [comment]
I can verify this on two Pixel 2 devices (mine and girldfriend). We had Nexus 5x before and it those have been on 8.1 as well and worked like charm with K9. Since we are using the pixel 2 the sync is inconsistent in the way Joe explained. Fully closing K9 and restarting fixes it temporarily, but this is not the way it should be.

**Comment by lldaedalusll** [comment]
It seems to be related to: https://github.com/k9mail/k-9/issues/3167

**Comment by nyetwurk** [comment]
Pixel 2, 8.1.0, optimize off, dovecot server
Same issue. No suggestions above work.

**Comment by lldaedalusll** [comment]
Disabling battery optimizations for k9 mail made it sync correctly again (see #3167). Had the same thing disabled on my nexus 5x, but did not remember.

**Comment by JoeT1969** [comment]
Battery Optimization fixed it.  I turned it off on my Pixel 2 XL and
K-9Mail has been fine ever since.  Thanks!!

**Comment by Cindy62707** [comment]
Samsung S7 Edge:  K9 will not sync on one of my accounts.   It may randomly sync once a day even though it is set to sync every 15 minutes.  Manual sync will not sync new emails either.  I cannot get any new emails on this account today but if I go on my computer out to the ISP site I have plenty of new emails.   K9 is set to use all the battery it wants,  I've tried Polling All folders and also just 1rst Class folders.  I've changed the sync time from every 15 minutes to every 10.  I've deleted and readded the account.  I've rebooted my phone.  Nothing helps.  The other accounts do not sync as scheduled but they will sync with a manual sync.  Settings on this account compared to the other accounts are identical (including the same ISP on 3 other accounts).  I've reported the problem on feedback several times.  Just like I've reported that K9 crashes most of the time whenever I try to delete any emails from this account.  No reply from the developers at all.  If they don't care to fix the problem, then I need to start looking for a different email app.  Can anyone suggest a good one for Android phones?  Or can anyone tell me how to fix this annoying problem?  I need access to my emails when I am away from home.  Thank you!
